runatlantis.io/docs/command-requirements.md

```diff
@@ -1,9 +1,9 @@
-# Apply Requirements
+# Command Requirements
```

Comment:
Hm the renaming of this file will remove any existing links that point to this page.

There's a way to alias pages. If you keep this file rename in place, could you add a link for the original page apply-requirements.md to redirect to the new page?

Reply:
I added an alias page. I will try redirect plugin later.

https://vuepress-community.netlify.app/en/plugins/redirect/

Reply:
cc: @chenrui333 @lilincmu in case this is what you folks mentioned a couple months ago